.github/workflows/main.yml

```diff
@@ -113,7 +113,16 @@ jobs:
           file: ./coverage.lcov
           flag-name: lcov-${{ matrix.DB }}-py${{ matrix.python-version }}
           parallel: true
-
+          
+  ruff:
```

Comment:
Hi @nokeechia , 

Thanks for contributing to this project, 

Had some minor request for changes in this PR, We were planning to move away from flake8 so for that,
1. Can you rename this job as `lint` instead of `ruff`, and add black action step from `test-sqlachemy-history` job + remove the falke8 check.
2. Can you also add ruff in dev dependency in pyproject.toml and remove flake8 from there too.

Thanks. 👍